packages/design-system/src/components/FieldContainer/FieldContainer.tsx

```diff
@@ -12,7 +13,15 @@ interface FieldContainerProps {
    * The HTML element used to render the container
    */
   component: 'div' | 'fieldset';
+  /**
+   * A unique ID to be used for the error message. If one isn't provided, a unique ID will be generated.
+   */
+  errorId?: string,
```

Comment:
The errorId is handled the same as all the other ids, you can provide one, and if not, a unique one will be generated